app/views/admin/branches/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container mt-4">
+  <%= render 'shared/notice' %>
+  <div class="border p-4">
+    <div class="d-flex justify-content-between align-items-center mb-4">
+      <h2><%= link_to "Branches", branches_path, class: "text-decoration-none" %> / <%= @branch.name %> / Orders</h2>
+    </div>
+    <% if @orders.blank? %>
+      <div class="alert alert-dark mx-5" role="alert">
+        <strong>No orders for this branch are present.</strong>
+      </div>
+    <% else %>
+      <table class="table table-striped">
```

Comment:
We should cart orders to admin.
pls discuss with @himanshumishra31 sir, whether we should add filters here or not?